azure-pipelines-external.yml

```diff
@@ -20,9 +20,9 @@ stages:
       steps:
       - checkout: self
       - task: NuGetToolInstaller@1
-        displayName: 'Use NuGet 5.8.1'
+        displayName: 'Use NuGet 6.2'
         inputs:
-          versionSpec: '5.8.1'
+          versionSpec: '6.2.x'
```

Comment:
Updated to the latest recommended version.

Using `x` as a wildcard, as per the [NuGetToolInstaller docs](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/nuget?view=azure-devops).

Specifying a two-part version means we will automatically pick up security patches.
We could decide to use a one-part version (e.g. `6.x`), although that might increase the risk of inadvertently picking up undesired behavioural changes.